clients/web/src/views/layout/ProgressListView.js

```diff
@@ -17,6 +17,9 @@ var ProgressListView = View.extend({
     initialize: function (settings) {
         this.eventStream = settings.eventStream;
         this.eventStream.on('g:event.progress', this._handleProgress, this);
+        // if the event stream disconnects, clear the progress so we don't have
+        // stale values lingering.
+        this.eventStream.on('g:eventStream.stop', this._clearProgress, this);
```

Comment:
Use `this.listenTo` to prevent memory leaks, in case the ProgressListView gets destroyed before the `EventStream` object.